src/store/getters/state-getters.ts

```diff
@@ -203,17 +203,16 @@ export const getStaffPayment = (state: any): any => {
   return state.stateModel.staffPayment
 }
 
+/** The staff payment validity. */
+export const isStaffPaymentValid = (state: any): any => {
```

Comment:
Please set return type.

Reply:
Not needed as this is now part of isFilingValid